### PR TITLE
adjust x-axis title

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR proposes to add an informative title for the plots. Might need an adjustment for the option to display absolute values via the [API](https://github.com/asreview/asreview-insights#example-relative-versus-absolute-axes )

### CODE REVIEW

File: asreviewcontrib/insights/plot.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def _plot_recall(ax, labels, x_absolute=False, y_absolute=False):
 
     ax.step(x, y, where='post')
     ax.set_title("Recall")
-    ax.set(xlabel='#', ylabel='Recall')
+    ax.set(xlabel='Proportion total number of records', ylabel='Recall')
```
Comment (J535D165) [inline asreviewcontrib/insights/plot.py:73]: This label should be dependent on argument `x_absolute`. If you come up with a description for the absolute axis, then I will implement the switch.

**Review by J535D165 — COMMENTED** [review]
Thanks

File: asreviewcontrib/insights/plot.py (hunk 1 above)
Comment (Rensvandeschoot) [reply asreviewcontrib/insights/plot.py:73]: I expected such a question :-) 
What about "Number of records"?

Comment (Rensvandeschoot) [inline asreviewcontrib/insights/plot.py:73]:
```suggestion
    ax.set(xlabel='Proportion of reviewed records', ylabel='Recall')
```

File: asreviewcontrib/insights/plot.py [hunk 2]
```diff
@@ -267,7 +267,7 @@ def _plot_erf(ax, labels, x_absolute=False, y_absolute=False):
 
     ax.step(x, y, where='post')
     ax.set_title("ERF")
-    ax.set(xlabel='#', ylabel='ERF')
+    ax.set(xlabel='Proportion total number of records', ylabel='ERF')
```
Comment (Rensvandeschoot) [inline asreviewcontrib/insights/plot.py:270]:
```suggestion
    ax.set(xlabel='Proportion of reviewed records', ylabel='ERF')
```

**Comment by J535D165** [comment]
Fixed in 8ef6af591c0aa568dc45a8828a7eacf3244b2f2e

**Pull request closed without merging**